conda_rpms/generate.py

```diff
@@ -48,6 +49,14 @@ def render_dist_spec(dist, config):
                                 rpm_prefix=rpm_prefix,
                                 install_prefix=install_prefix)
 
+def render_env(branch_name, label, tag):
+    env_info = {'url': 'http://link/to/gh',
+                'name': branch_name,
+		'label' : label,
+                'summary': 'A SciTools environment.',
+                'version': '1',}
```

Comment:
We talked in person, and went for the `number of commits into the repo` approach. Unit testing this will be really helpful, but I understand if that makes it much harder to develop currently (whilst we wait on Bill's testing PR).